aws/logs_monitoring/tools/integration_tests/snapshots/cloudwatch_log_lambda_invocation.json~snapshot

```diff
@@ -379,19 +379,19 @@
                 "start": "1583425836164691700",
                 "duration": "458278320",
                 "meta": {
-                  "account_id": "0",
```

Comment:
@nhinsch for later -- probably need to sort the keys of the json snapshot to avoid the flakiness?